### PR TITLE
Fixing problems in logs and small tweaks

### PR DESCRIPTION
**Pull request by gps2601**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const app = require('./app');
 const PORT = process.env.PORT || 3000;
 const mongoose = require('mongoose');
 
-mongoose.connect(process.env.MONGODB_URI);
+mongoose.connect(process.env.MONGODB_URI,  { useNewUrlParser: true });
 
-app.listen(PORT, () => console.log(`Your app is listening on port ${PORT}`));
+app.listen(PORT, () => console.log(`Your app is listening on port ${PORT}\nView it's awesomeness here: http://localhost:${PORT}`));
```
Comment (peterjcole) [inline server.js]: Small typo, should be ```its```

Comment (peterjcole) [reply server.js]: Sorry if that's a bit pedantic, looks good though!

**Review by peterjcole — APPROVED** [review]
:spiral_notepad:

File: server.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const app = require('./app');
 const PORT = process.env.PORT || 3000;
 const mongoose = require('mongoose');
 
-mongoose.connect(process.env.MONGODB_URI);
+mongoose.connect(process.env.MONGODB_URI,  { useNewUrlParser: true });
```
Comment (AnnaWeakclaw) [inline server.js:8]: Addresses the warning on deprecated parser

File: tests/database.test.js [hunk 3]
```diff
@@ -32,7 +35,7 @@ describe('One entry in Database', () => {
     };
     const newBin = new Bin(binJson);
     await newBin.save();
-    const count = await Bin.count();
+    const count = await Bin.countDocuments();
```
Comment (AnnaWeakclaw) [inline tests/database.test.js:38]: Count() is deprecated

**Review by AnnaWeakclaw — APPROVED** [review]
👍

**Review by HelenHTech — APPROVED** [review]
Looks great. Thanks Geoff!

**Pull request merged**